pre-commit/action.yaml

```diff
@@ -18,10 +21,26 @@ runs:
       with:
         token: ${{ inputs.token }}
 
+    - name: Set target files
+      id: set-target-files
+      run: |
+        if [ -z "${{ inputs.base-branch }}" ]; then
+          echo "option=--all-files" >> $GITHUB_OUTPUT
+          exit 0
+        fi
+        target_files=$(git diff --name-only "${{ inputs.base-branch }}"...HEAD)
```

Comment:
Is this equivalent to this command? Then it might be simpler.

```bash
pre-commit run --from-ref "${{ inputs.base-branch }}" --to-ref HEAD
```

```
  --from-ref FROM_REF, --source FROM_REF, -s FROM_REF
                        (for usage with `--to-ref`) -- this option represents the original ref in a `from_ref...to_ref` diff expression. For `pre-push` hooks, this represents the branch you are pushing to. For
                        `post-checkout` hooks, this represents the branch that was previously checked out.
  --to-ref TO_REF, --origin TO_REF, -o TO_REF
                        (for usage with `--from-ref`) -- this option represents the destination ref in a `from_ref...to_ref` diff expression. For `pre-push` hooks, this represents the branch being pushed. For
                        `post-checkout` hooks, this represents the branch that is now checked out.
```

Reply:
Yes. I didn't notice that option. Updated https://github.com/autowarefoundation/autoware-github-actions/pull/237/commits/a6bd4982c27b08ac60a402e0b02ca1d5b137d86c.